integration-cli/docker_cli_network_unix_test.go

```diff
@@ -996,26 +1001,23 @@ func (s *DockerNetworkSuite) TestDockerNetworkHostModeUngracefulDaemonRestart(c
 	testRequires(c, DaemonIsLinux, NotUserNamespace)
 	s.d.StartWithBusybox()
 
-	// Run a few containers on host network
-	for i := 0; i < 10; i++ {
-		cName := fmt.Sprintf("hostc-%d", i)
-		out, err := s.d.Cmd("run", "-d", "--name", cName, "--net=host", "--restart=always", "busybox", "top")
-		c.Assert(err, checker.IsNil, check.Commentf(out))
-	}
+	// Run a container on host network
+	out, err := s.d.Cmd("run", "-d", "--name=hostc", "--net=host", "--restart=always", "busybox", "top")
```

Comment:
why just run a container? I think we should make sure it work when running a few containers.